pkg/aws/actuator/actuator.go

```diff
@@ -54,6 +54,10 @@ const (
 	roAWSCredsSecret          = "cloud-credential-operator-iam-ro-creds"
 	openshiftClusterIDKey     = "openshiftClusterID"
 	clusterVersionObjectName  = "version"
+
+	secretDataAccessKey      = "aws_access_key_id"
+	secretDataSecretKey      = "aws_secret_access_key"
+	secretDataCredentialsKey = "credentials"
```

Comment:
There were some comments on one of our calls that you will need a second config file to point to this creds file, or possibly the section within? Did we determine if that was necessary or not?

Reply:
I tried a couple of images that I and @cuppett made covering Python, Golang, PHP, and Java. I've seen all four work with the same credentials configuration file (although not every language uses the same environment variables or setup).

golang:  `AWS_SDK_LOAD_CONFIG=true AWS_SHARED_CREDENTIALS_FILE=<path to creds/config>`
python: `AWS_SHARED_CREDENTIALS_FILE=<path to creds/config>`
java and php couldn't be pointed to a non-default location so i had to: `mkdir -p ${HOME}/.aws && cp <path to credentials/config> ${HOME}/.aws/`